src/DemoUI/src/Views/resources/css/index.css

```diff
@@ -0,0 +1,1865 @@
+    box-sizing: content-box;
+}
+
+.pl-configure-shipping-method-wrapper td {
+    padding: 0 !important;
+    box-sizing: border-box;
+    background-color: #f5f5f5 !important;
+    position: relative;
+    z-index: 200;
+    animation: pl-fade-in 0.5s ease;
+    border: 1px solid #2095F2 !important;
+}
+
+.pl-configure-shipping-method-form-wrapper {
+    padding: 20px 20px 0 20px !important;
+    border-bottom: 1px solid #dcdcdc;
+}
+
+.pl-configure-shipping-method-form-wrapper:after,
+.pl-configure-shipping-method-form-wrapper:before {
+    content: '';
+    display: block;
+    position: absolute;
+    left: 100%;
+    width: 0;
+    height: 0;
+    border-style: solid;
+    z-index: 1 !important;
+}
+
+.pl-configure-shipping-method-form-wrapper:after {
+    top: -19px;
+    left: 15%;
+    border-color: transparent transparent #f5f5f5 transparent;
+    border-width: 10px;
+}
+
+.pl-configure-shipping-method-form-wrapper:before {
+    top: -22px;
+    left: calc(15% - 1px);
+    border-color: transparent transparent #2095F2 transparent;
+    border-width: 11px;
+}
+
+.pl-shipping-method-form {
+    padding: 0 !important;
+}
+
+.pl-form-section-wrapper {
+    padding: 0 0 30px 0 !important;
+    box-sizing: border-box;
+    width: 49%;
+    display: inline-table;
+}
+
+.pl-shipping-method-form > div:nth-child(1) {
+    padding-right: 10px !important;
+}
+
+.pl-shipping-method-form > div:nth-child(2) {
+    padding-left: 10px !important;
+}
+
+.pl-form-section-title-wrapper {
+    padding: 0 !important;
+    height: 20px !important;
+    line-height: 20px !important;
+    margin-bottom: 20px;
+}
+
+.pl-configure-shipping-method-button-wrapper {
+    text-align: right;
+    padding: 10px 20px !important;
+}
+
+.pl-configure-shipping-method-button-wrapper button {
+    margin: 0 0 0 15px !important;
+}
+
+.pl-form-section-title-wrapper {
+    display: flex;
+}
+
+.pl-form-section-title {
+    height: 20px;
+    line-height: 20px;
+    font-size: 16px;
+    color: #5d5d5d;
+}
+
+.pl-form-section-title-line {
+    height: 20px;
+    padding-top: 10px;
+    padding-left: 10px;
+    flex-grow: 1;
+}
+
+.pl-form-section-title-line hr {
+    border: 1px solid #dcdcdc;
+    margin: 0 !important;
+    padding: 0 !important;
+}
+
+.pl-form-selection-help-button-wrapper {
+    margin: 0 !important;
+    padding: 0 !important;
+    height: 20px;
+    width: 20px;
+    text-align: center;
+    line-height: 20px;
+    color: #2095F2;
+    border: 1px solid #2095F2;
+    border-radius: 50%;
+    cursor: pointer !important;
+}
+
+.pl-tooltip {
+    position: relative;
+    display: inline-block;
+}
+
+.pl-tooltip:hover .pl-tooltip-concrete {
+    visibility: visible !important;
+    opacity: 1 !important;
+}
+
+.pl-tooltip-concrete {
+    box-sizing: border-box;
+    width: 140px;
+    padding: 15px;
+    position: absolute;
+    background-color: white;
+    color: #5d5d5d;
+    font-size: 12px !important;
+    line-height: 14px !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)

Reply:
This file is deleted (replaced) in the next PR.